README.md

```diff
@@ -1,22 +1,54 @@
-# kubescape.io website
+# kubescape.io Website
 
-This repository contains the source files for the kubescape.io website.
+This repository contains the source files for the [kubescape.io](https://kubescape.io) website. The website is built using [mkdocs](https://www.mkdocs.org/) with the [mkdocs-material](https://squidfunk.github.io/mkdocs-material/) theme.
 
-## Building the site
+## Table of Contents
 
-### Local testing
+- [Getting Started](#getting-started)
+- [Local Testing](#local-testing)
+- [Production](#production)
+- [Contributing](#contributing)
+- [License](#license)
 
-We use [mkdocs](https://www.mkdocs.org/) with the [mkdocs-material](https://squidfunk.github.io/mkdocs-material/) theme. 
+## Getting Started
 
-To build and test locally: 
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
 
-* [install mkdocs-material](https://squidfunk.github.io/mkdocs-material/getting-started/) with `pip` or Docker 
-* run `mkdocs serve`
+### Prerequisites
 
-## Local build
+- Python installed on your local machine
+- pip 
 
-Run `mkdocs build`, and the site will be built in the `site/` directory.
+### Setting Up
 
-### Production
+1. Clone the repository to your local machine.
+```bash
+git clone https://github.com/kubescape/website.git
+```
+2. Navigate to the project directory.
+```bash
+cd website
+```
+3. Install the required dependencies using the `requirements.txt` file.
+```bash
+pip install -r requirements.txt
+```
 
-The site is hosted with thanks to Netlify.  On each successful push to the repository, the site will be published by their bot.
\ No newline at end of file
+## Local Testing
+
+To build and test the website locally, follow these steps:
+
+1. Naviagte to project directory and run the command `mkdocs build`. The site will be built in the `site/` directory.
+2. After building the website locally, you can run the website locally using the command `mkdocs serve`.
+
+## Production
+
+The site is currently hosted with thanks to Netlify. On each successful push to the repository, the site will be published by their bot.
+
+## Contributing
+
+We welcome contributions from the community. Please read our [contributing guide](https://github.com/kubescape/kubescape.io/blob/main/docs/project/contributing.md) for more information.
+
+## License
```

Comment:
The license is already prominently displayed on GitHub and a LICENSE file is included.

Reply:
Makes sense. Removing the License section then, Sound good?